### PR TITLE
Fix macos apple install command

### PR DESCRIPTION
**Pull request by Machi3mfl**
### Description
This PR resolves issue #5466

### CODE REVIEW

**Comment by Desvelao** [comment]
# Wazuh dashboard

Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

## UI

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| The command in the deployment guide to starting a new macOS agent must use `sudo` | :gree_circle: | :green_circle: | :black_circle: |

**Details**
<details>
<summary>:green_circle: The command in the deployment guide to starting a new macOS agent must use `sudo`</summary>

Chrome - :green_circle:
image

Firefox - :green_circle:
image

Safari - :black_circle:

</details>

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,15 @@
...
 All notable changes to the Wazuh app project will be documented in this file.
 
+## Wazuh v4.4.3 - OpenSearch Dashboards 2.6.0 - Revision 01
+
+### Added
+
+- Support for Wazuh 4.4.3
+
+### Fixed
+- Fix macos apple install command [#5466](https://github.com/wazuh/wazuh-kibana-app/pull/5466)
```
Comment (Desvelao) [inline CHANGELOG.md:12]: **nitpick**: add a new line after `### Fixed`

**Review by asteriscos — APPROVED** [review]
LGTM :heavy_check_mark:

**Review by Desvelao — APPROVED** [review]
**review**

code :green_circle: 
test :green_circle: https://github.com/wazuh/wazuh-kibana-app/pull/5470#issuecomment-1557195364

**Pull request merged**